### DISCUSSION TITLE
Welcome to swiss-gc Discussions!

### DISCUSSION THREAD

**Discussion by emukidid** [emukidid/swiss-gc#508] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Heyman101** [comment]
Hello sir, sorry for being absolute noob to GameCube modding, I’ve modded most other systems though. Anyways I have an action replay version 1.08 so I need to run sdload, when I try to patch the memory card it says obtaining low level access to drive....failed. I am not sure why? Any help would be great. Sorry again for dumb questions but I’ve been around the internet for days now. Thanks a lot and thanks for creating the things you have!!!!!

**Reply by Extrems** [reply]
Are you following the instructions [here](https://www.gc-forever.com/wiki/index.php?title=SD_load#Step_1_.5BSetting_up_the_SD_Card_-_Windows_Vista_.2F_7_.2F_8_.2F_10.5D)? You need administrator privileges.

**Reply by Heyman101** [reply]
I went into settings on the memory card after I formatted to FAT and set compatibility to Windows XP sp2, and clicked run as administrator, is that correct?

**Reply by Heyman101** [reply]
Also thanks to you as well for creating the sdload and everything else man, thanks!

**Reply by Extrems** [reply]
Uh, no, that's not what the instructions say. You need the Windows 10 compatible version available [here](https://www.gc-forever.com/forums/viewtopic.php?f=39&t=4540).

**Reply by Heyman101** [reply]
Trying to get into gc-forever, I’ve been looking for that zip file sir!!!! Geez I’m lame 😞, I was in the forums reading, just not that page you linked. Thanks for responding

**Reply by Heyman101** [reply]
I was just trying everything, I got that patch to work with your help, I’ll try to do the rest later, I really appreciate the link and response, have a good day guys, I’m sure I’ll be around, thanks again!

**Comment by wgmcgovern** [comment]
Hello Good Sir
I have a question regarding Swiss Updates. I am currently running Swiss version 1290 on my GC loader and noticed that some later versions of Swiss had Bricked the GC loader for the Game cube. My question is can I safely now update the swiss version 1290 to the latest version without causing any  bricking issues on my device? Any suggestions would be most appreciated. Thank You so much in advance for all your hard work that you do to constantly improve Swiss for the community.

Sincerely

Bill

**Reply by emukidid** [reply]
It was not Swiss that bricked these but rather the GC-Loader firmware updates: https://github.com/danielkraak/GC-Loader/blob/master/Firmware%20Updates/changelog.md

Just make sure yours is genuine.

**Comment by wgmcgovern** [comment]
Thank you so much for the info and the download link.Sent via the Samsung Galaxy S23+ 5G, an AT&T 5G smartphone
-------- Original message --------From: emukidid ***@***.***> Date: 9/22/25  3:00 AM  (GMT-05:00) To: emukidid/swiss-gc ***@***.***> Cc: wgmcgovern ***@***.***>, Comment ***@***.***> Subject: Re: [emukidid/swiss-gc] Welcome to swiss-gc Discussions! (Discussion #508) 
It was not Swiss that bricked these but rather the GC-Loader firmware updates: https://github.com/danielkraak/GC-Loader/blob/master/Firmware%20Updates/changelog.md
Just make sure yours is genuine.
